Command/RequestPasswordResetHandler.php

```diff
@@ -46,10 +46,6 @@ public function handle(RequestPasswordDataTransferObject $dataTransferObject)
     {
         $user = $this->userRepositoryCollection->findUserByUserName($dataTransferObject->userName);
 
-        if ($user === null) {
-            throw new EntityNotFoundException();
```

Comment:
Als we zo gaan beginnen dan moet die `throws` tag ook in `requestAction()` komen enzovoort